data-serving/scripts/prune-uploads/prune_uploads.py

```diff
@@ -0,0 +1,105 @@
+# prune_uploads.py -- Mark new uploads as ingested and delete failed uploads
+
+# This is a script that should be run periodically that sets every upload
+# older than the last successful upload to list = false (for non-UUID
+# sources). For UUID sources, set everything to list = true. After marking,
+# this deletes everything with list = false.
+
+import os
+import sys
+import pymongo
```

Comment:
nitpick and not at all a blocker, but `import pymongo` should go in its own group below the other import statements, separated by a blank line https://www.python.org/dev/peps/pep-0008/#imports